app/views/reviews/edit.html.erb

```diff
@@ -0,0 +1,14 @@
+<h2>Edit Review</h2>
+<%= form_tag "/shelters/#{@review.shelter.id}/reviews/#{@review.id}", method: :patch do %>
+  <%= label_tag :title %>
+  <%= text_field_tag :title, @review.title %><br>
+  <%= label_tag :rating %>
+  <%= text_field_tag :rating, @review.rating %><br>
+  <%= label_tag :content %>
+  <%= text_field_tag :content, @review.content %><br>
+  <%= label_tag "Picture (optional)" %>
+  <%= text_field_tag :picture, @review.picture %><br>
+  <%= submit_tag "Save Review" %><br><br>
+<% end %>
+
+<h4><%= link_to "Go Back", :back %></h4>
```

Comment:
Did you test this link ? I think this will take you to /shelters/#{@review.shelter.id}/reviews, which I don't believe currently has a view set up